extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/BodyHandlerConfig.java

```diff
@@ -0,0 +1,55 @@
+package io.quarkus.vertx.http.runtime;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+/**
+ * Vert.x Web body handler configuration
+ */
+@ConfigGroup
+public class BodyHandlerConfig {
+
+    /**
+     * Whether the files sent using {@code multipart/form-data} will be stored locally.
+     * <p>
+     * If {@code true}, they will be stored in {@code quarkus.http.body-handler.uploads-directory} and will be made
+     * available via {@code io.vertx.ext.web.RoutingContext.fileUploads()}.
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean handleFileUploads;
```

Comment:
Several config properties uses "enable", should this one follow this convention. Like `enableFileUploads`?

Reply:
Not sure. `enableFileUploads = false` might sound like it is not possible to upload files to the given server. But even with `enableFileUploads = false`, the multipart requests are accepted. It only causes that the files are not stored locally and the `RoutingContext.fileUploads()` returns an empty collection.

Reply:
Let me add a sentence about the negative behavior to the JavaDoc

Reply:
+1